app/Http/Controllers/Api/V1/QuestionController.php

```diff
@@ -2,13 +2,70 @@
 
 namespace App\Http\Controllers\Api\V1;
 
-use App\Http\Controllers\Controller;
+use App\Models\Survey;
+use App\Models\Question;
 use Illuminate\Http\Request;
+use Illuminate\Http\JsonResponse;
+use App\Http\Controllers\Controller;
+use App\Http\Requests\QuestionRequest;
+
 
 class QuestionController extends Controller
 {
-    public function create()
+    /**
+     * Create a new question 
+     *
+     * @param  QuestionRequest $request
+     * @param  Survey $survey
+     * 
+     * @return  JsonResponse
+     */
+    public function store(QuestionRequest $request, Survey $survey ): JsonResponse
+    {
+        $survey->questions()->create($request->validated());
+        
+        return response()->json([
+            'status'  => 'success',
+            'message' => 'Question had been succesfully added to the survey.'
+        ]);
+    }
+
+     /**
+     * Create a new question 
+     *
+     * @param  QuestionRequest $request
+     * @param  Question $question
+     * 
+     * @return  JsonResponse
+     */
+    public function update(QuestionRequest $request, Question $question): JsonResponse
     {
-        return "here";
+        $question->update($request->validated());
+        
+        return response()->json([
+            'status'  => 'success',
+            'message' => 'Question had been succesfully updated.'
+        ]);
     }
-}
+
+    /**
+     * Delete de question and the options associated with this question
+     *
+     * @param  Question $question
+     * @param  Survey $survey
+     *
+     * @return JsonResponse
+     */
+    public function destroy(Survey $survey, Question $question)
+    { 
+        $question->load('options');
```

Comment:
Dit is niet nodig omdat je hieronder via de query builder de options ophaalt dit kan je niet eager loaden.